plugins/powerd/fu-plugin-powerd.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2016 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#define MINIMUM_BATTERY_PERCENTAGE_FALLBACK	10
+
+struct FuPluginData {
+	GDBusProxy		*proxy;		/* nullable */
+};
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+	fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
+        
+        return;
```

Comment:
you don't need a return from void

Reply:
Thank you, I went back and adjusted this.

Reply:
If it helps, do you want me to make you a committer on the main repo, and then you can push a wip/g2pop/powerd branch? if it's WIP we can scribble all over it and then when it's ready for review we can squash it into a sensible commit and merge.

Reply:
Thank you, right now though, we still have more foundational work to do. I think we'll try to make the new plugin functional while on a fork, and then push it to a branch on the main repo for you to review at a higher level.

Reply:
Okay, sounds like a plan. Yell if you need any help or advice.